src/component/navBar.js

```diff
@@ -0,0 +1,160 @@
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import InputBase from '@mui/material/InputBase';
+import Button from '@mui/material/Button';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import SearchIcon from '@mui/icons-material/Search';
+import { useState, useEffect } from 'react';
+
+const Search = styled('div')(({ theme }) => ({
+  marginLeft: 'auto',
+  display: 'flex',
+  alignItems: 'center',
+  borderRadius: theme.shape.borderRadius + 20,
+  backgroundColor: 'rgba(229, 229, 229, 1)',
+  paddingLeft: theme.spacing(2),
+  paddingRight: theme.spacing(1),
+}));
+
+const StyledInputBase = styled(InputBase)(({ theme }) => ({
+  color: 'rgba(0, 0, 0, 0.87)',
+  '& .MuiInputBase-input': {
+    padding: theme.spacing(1),
+    transition: theme.transitions.create('width'),
+    width: '12ch',
+    '&:focus': {
+      width: '20ch',
+    },
+  },
+}));
+
+export default function SearchAppBar() {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [genre, setGenre] = useState('');
+  const [records, setRecords] = useState([]);
+
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleGenreChange = (event) => {
+    const selectedGenre = event.target.innerText;
+    console.log('Selected genre:', selectedGenre);
+    setGenre(selectedGenre);
+    handleClose();
+  };
+
+  useEffect(() => {
+    if (genre) {
+      console.log('Fetching movies for genre:', genre);
+      fetch(`http://localhost:3000/movies/genre/${genre}`)
+        .then((response) => {
+          if (!response.ok) {
+            throw new Error('Network response was not ok');
+          }
+          return response.json();
+        })
+        .then((data) => {
+          console.log('Movies fetched:', data);
+          setRecords(data);
+        })
+        .catch((err) => console.log('Fetch error:', err));
+    }
+  }, [genre]);
+
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <AppBar
+        position="static"
+        sx={{ backgroundColor: '#ffffff', width: '100%' }}
+      >
+        <Toolbar sx={{ display: 'flex', justifyContent: 'space-between' }}>
+          <Typography
+            variant="h6"
+            noWrap
+            component="a"
+            href="http://localhost:3001"
+            sx={{
+              color: 'rgba(65, 140, 251, 1)',
+              fontFamily: 'Rubik',
+              fontSize: '26.76px',
+              fontWeight: 700,
+              lineHeight: '31.72px',
+              textAlign: 'left',
+              textDecoration: 'none',
+              marginRight: 4,
+            }}
+          >
+            MMDB
+          </Typography>
+          <Typography
+            variant="body1"
+            noWrap
+            component="a"
+            href="http://localhost:3001"
```

Comment:
Try redirecting here to just / it will redirect to the root page of this website. This will help in reducing the logic needed when deploying this website into different environments.